project_task_project_mandatory/README.rst

```diff
@@ -0,0 +1,48 @@
+Enhancements related to Project Management.
+
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Project Task Project Mandatory
+==============================
+
+This module extends the functionality of project to support to make project required at task.
```

Comment:
This module extends the functionality of project to make it mandatory in the tasks.